nuxt.config.js

```diff
@@ -34,8 +34,13 @@ export default defineNuxtConfig({
     buildAssetsDir: '/nuxt/',
     head: {
       title: 'Centrapay Docs',
-      link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }]
-    }
+      link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }],
+      meta: [
+        // This is needed to send the entire URL of the page in Freshdesk support tickets
+        // Refer to https://community.freshworks.com/freshdesk-11246/where-does-it-come-from-23026
+        { name: 'referrer', content: 'no-referrer-when-downgrade' }
```

Comment:
Preference here would be to prefill the Page Title + URL as custom fields in the form - https://developers.freshdesk.com/widget-api/#:~:text=With%20the%20Freshdesk%20help%20widget,customers%20open%20the%20contact%20form.

I think this is ok for now but this is something to be wary of e.g. if we ever supported user account - https://developer.mozilla.org/en-US/docs/Web/Security/Referer_header:_privacy_and_security_concerns